new/app/appStylesheet.ts

```diff
@@ -101,7 +101,8 @@ setStyleBuilders({
     baseButton: (config) => {
         return {
             backgroundColor: config.themeColorPalette.secondaryBackground,
-            padding: 7
+            padding: 7,
+            justifyContent: "center",
```

Comment:
why need to justify content? The width of baseButton should be same as its content plus padding

Reply:
removed, it was used to test but forgot to del

Reply:
You could raise a ticket to fix it later on